### ISSUE TITLE
Codebook

### ISSUE THREAD

**Issue by gsilvapt** [drazisil-org/dashboard-zero#14]
As it has been recommended in Data Science forums, I think it would be great to create a codebook that aggregates all steps and standards we have been using.
This also mean that future contributors will not change the structure drastically and things will not get completely different organizations in the future.

Another great advantage of this "book" is that more experienced contributors can have a lot and propose suggestions to improve the code - either for fetching the data and/or crunching numbers.

**Comment by drazisil** [comment]
I'm not sure what that is, but sounds good. If you info from let me know.

**Comment by gsilvapt** [comment]
https://d396qusza40orc.cloudfront.net/getdata/lecture_slides/01_03_componentsOfTidyData.pdf

Slides 5 and 6

**Comment by drazisil** [comment]
Ah, ok. I can do that. I'm not really treating the data from github to the csv so far, except now is_pull_request is a culculated field.

**Comment by drazisil** [comment]
@gsilvapt I think using SQL to describe the formulas would be best, unless the industry standard is to use R.

**Comment by gsilvapt** [comment]
No, that is not what I mean. SQL seems easier to "read", even for non-power users. However, I am not that comfortable with SQL, so I cannot help you with that. Thus my suggestion to make it simpler and just write it in simpler terms. Of course, if we want to keep it in SQL, it's fine by me. One other chance for me to learn SQL :+1:

**Comment by drazisil** [comment]
@gsilvapt SQL is a good skill to know when doing metrics :)

**Comment by gsilvapt** [comment]
I know the basic, not complex stuff :)

**Comment by drazisil** [comment]
@gsilvapt did you delete the codebook, or just hide it on me?

**Comment by gsilvapt** [comment]
I did unintentionally. I forgot it was under the old data tab. Is there a way to retrieve it from older versions? You shouldn't have pulled my repo, it was not ready yet - forgot to add that in the comments, sorry

**Comment by drazisil** [comment]
Heh, no worries. Unlike the code I wiped out (twice) because I kept running git stash, we can. I actually already have :-)

**Comment by gsilvapt** [comment]
Well, I was preparing a different repository in my own one, and was using your codebook for guidance. Latter, whenever mine was ready, I would take care of the codebook and copy-paste some of your texts over there. But forgot to add that in the comments and/or in the issues, thus the mess :P Sorry and thank you for getting it back :D

**Comment by drazisil** [comment]
No worries. I just wanted it handy for reference. You can do it any way you want :)

**Comment by gsilvapt** [comment]
Okay, so I managed to retrieve the document from my trash-can. 
I'll let it stay in my repo, if you or anyone needs it - I'm not sure if you can pull just one single item. If so, be my guess.

Nevertheless, I will do the proper changes anytime soon.

**Comment by drazisil** [comment]
Ah, I restored it here: https://github.com/drazisil/dashboard-zero/blob/jucy-stats/docs/codebook.md I forgot to mention I changed the path, sorry.

**Comment by drazisil** [comment]
This is part of documentation. I'm not sure the format it will take, and what information it will contain, but it's still a good plan.

**Comment by drazisil** [comment]
I don't think this is needed (ie: programmers won't care), but let me know if you want to work on it.
